bin/dbsync

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+var path   = require('path');
+var fs     = require('fs');
+var coffee = require('coffee-script');
+var src    = path.join(path.dirname(fs.realpathSync(__filename)), '../src');
+
```

Comment:
Whether it is gulp or not I was thinking more in line of a developer script and to rerun linting automatically. I did not intend that linting would happen when it runs as a standalone.